GhostNetwork.Content.Api/Models/UpdateMediaModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace GhostNetwork.Content.Api.Models;
+
+public class UpdateMediaModel
+{
+    public Guid Id { get; set; }
```

Comment:
What's the use case for Id?
Af far as I see you are doing full update for media field in database, so `media.id` looks redundant

Reply:
Do we really need it?